lua/toggleterm/config.lua

```diff
@@ -65,6 +69,9 @@ local config = {
     winblend = 0,
     title_pos = "left",
   },
+  responsiveness_settings = {
```

Comment:
I think the addition of the term `_settings` is verbose, it's already in a config object so I don't thing that's needed. Trying to think of how things like css or desktop apps refer to their layout/responsive features, `responsiveness` on it's own is ok in lieu of something better